samples/app-dev/API-Development/REST/rest-ssl-certs-k8s-secret/readme.md

```diff
@@ -0,0 +1,212 @@
+					secretKeyRef:
+					  name: client-secret
+					  key: tls.crt
+				- name: client_key
+				  valueFrom:
+					secretKeyRef:
+					  name: client-secret
+					  key: tls.key
+				- name: FLOGO_APP_PROPS_ENV
+				  value: auto
+			  ports:
+				- containerPort: 9999
+			  volumeMounts:
+				- name: tls-persistent-storage
+				  mountPath: /var/lib/misc
+		  volumes:
+			- name: tls-persistent-storage
+			  emptyDir: {}
+
+	```
+
+Apply this file using the following command;
+	`kubectl apply -f deployment-env.yaml`
+
+b. Pod Configuration File (Volume Mount Approach)
+
+The volume-pod.yaml file demonstrates how to use volume mounts to access the secrets. 
+
+###Volume Mounts Configuration for YAML
+
+Below is a generic YAML configuration for deploying an application with SSL certificate support using volume mounts.
+
+	```yaml
+
+	apiVersion: v1
+	kind: Pod
+	metadata:
+	  name: mtls-volume
+	spec:
+	  containers:
+		- name: mtls-volume-container
+		  image: restserviceclientapp:1.1.2
+		  imagePullPolicy: Never
+		  env:
+			- name: FLOGO_APP_PROPS_ENV
+			  value: auto
+		  volumeMounts:
+			- name: server-secret-volume
+			  mountPath: /etc/server-secret-volume
+			  readOnly: true
+			- name: client-secret-volume
+			  mountPath: /etc/client-secret-volume
+			  readOnly: true
+	  volumes:
+		- name: server-secret-volume
+		  secret:
+			secretName: server-secret
+		- name: client-secret-volume
+		  secret:
+			secretName: client-secret
+	```		
+				
+Apply this file using the following command;
+	`kubectl apply -f volume-pod.yaml`	
+
+
+6. Run the App
+
+Deploy the app using the specified configuration files. The Docker image will be automatically used in the pods. Verify the deployment by checking the logs and endpoints.
+
+Example Commands for Debugging
+
+Check pod status:
+	`kubectl get pods`
+
+View logs:
+	`kubectl logs <pod-name> -c <container-name>`
+
+Execute commands inside the pod:
+	`kubectl exec -it <pod-name> -- /bin/sh`
+
+![Check pod status and logs](../../../import-screenshots/rest-ssl-certs-k8s-secret/6.png)
+
+##Notes
+
+Ensure your Kubernetes cluster has sufficient resources for deployment.
+
+Update certificate paths and secret names in the configuration files as required.
+
+By following these steps, you can successfully configure and deploy a REST service and client with SSL certificate support in Kubernetes.
+
+## Contributing
+If you want to build your own activities for Flogo please read the docs here.
+
+If you want to showcase your project, check out [tci-awesome](https://github.com/TIBCOSoftware/tci-awesome)
+
+You can also send an email to `tci@tibco.com`
+
+## Feedback
+If you have feedback, don't hesitate to talk to us!
+
+* Submit feature requests on our [TCI Ideas](https://ideas.tibco.com/?project=TCI) or [FE Ideas](https://ideas.tibco.com/?project=FE) portal
+* Ask questions on the [TIBCO Community](https://community.tibco.com/answers/product/344006)
+* Send us a note at `tci@tibco.com`
+
+## Help
```

Comment:
remove tci...we will put vscode user guide link for this section

Reply:
currently removed the whole section, will put vscode link once we have docs in place